src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx

```diff
@@ -425,12 +416,7 @@ export default class ArcanaTracking extends Analyser {
 
 			// Modify log, they were holding onto this card since index
 			// Differenciate depending on searchLatest
-			let arcanaStatus: number | undefined
-			if (this.lastDrawnBuff && this.crownPlays.includes(cardActionId)) {
-				arcanaStatus = this.lastDrawnBuff.data
-			} else {
-				arcanaStatus = this.arcanaActionToStatus(cardActionId)
-			}
+			const arcanaStatus: number | undefined = this.arcanaActionToStatus(cardActionId)
```

Comment:
is this due to the analyzer change? So we don't need to do worry about it being dropped from the lastDrawnBuff?

Reply:
I see the intention is to pick up statuses that were /statusoff or when the player died, but it seems this specific portion only applied for crown plays
I think the dropped-off statuses portions need to be rewritten a bit because it also doesn't seem to also be picking up death events appropriately

here's an example log with a death event that wasn't appropriately captured
https://xivanalysis.com/fflogs/L3mJVpA1rc4hGgKY/2/2
see timestamp 1:51 under arcana tracking
I think this is going to happen every time there is a death event when someone holds a card, but I could be wrong

when I have time, I'll look into the /statusoff event and see what we can do to improve this part